tests/scripts/run_performance.sh

```diff
@@ -17,7 +17,10 @@ run_perf_models_other() {
 
     env pytest models/demos/ttnn_falcon7b/tests -m $test_marker
 
-    env pytest models/demos/resnet/tests -m $test_marker
+    # Separate calls since we can't mix switching between number of cqs
+    env pytest models/demos/resnet/tests/test_perf_resnet.py::test_perf_bare_metal -m $test_marker
```

Comment:
Sorry to do this to you... but could you separate this into separate tests or put it all in the same file 🥴

Reply:
Separated into separate files